sdk/dns/Azure.ResourceManager.Dns/CHANGELOG.md

```diff
@@ -1,12 +1,59 @@
 # Release History
 
-## 1.0.0-preview.2 (Unreleased)
+## 1.0.0-beta.1 (Unreleased)
 
+### Breaking Changes
 
-## 1.0.0-preview.1 (2020-09-23)
+New design of track 2 initial commit.
```

Comment:
Consider to provide some major changes of new track2:

### General New Features

- Support MSAL.NET, Azure.Identity is out of box for supporting MSAL.NET
- Support [OpenTelemetry](https://opentelemetry.io/) for distributed tracing
- HTTP pipeline with custom policies
- Better error-handling
- Support uniform telemetry across all languages

> NOTE: For more information about unified authentication, please refer to [Azure Identity documentation for .NET](https://docs.microsoft.com//dotnet/api/overview/azure/identity-readme?view=azure-dotnet)